rotkehlchen/db/settings.py

```diff
@@ -526,7 +532,19 @@ def get_query_retry_limit(self) -> int:
 
     def get_evm_indexers_order_for_chain(self, chain_id: ChainID) -> tuple[EvmIndexer, ...]:
         """Return chain-specific indexer order falling back to defaults."""
-        return self._evm_indexers_order_per_chain[chain_id]
+        order = self._evm_indexers_order_per_chain[chain_id]
+        if (override := self._evm_indexers_order_override_var.get()) is None:
+            return order
+
+        return (override,) + tuple(indexer for indexer in order if indexer != override)
```

Comment:
not sure if it's a needed optimisation but you can avoid creating a tuple when `override is not in `order`.

maybe `order` should be renamed to `default_order`